docker-compose.yml

```diff
@@ -5,46 +5,58 @@ services:
     image: 'postgres:16-alpine'
     restart: unless-stopped
     ports:
-      - '5432:5432' # Use a port unlikely to be in use so the example "Just Works"
+      - '5432:5432'
     environment:
       - POSTGRES_DB=indexer
       - POSTGRES_USER=indexer
       - POSTGRES_PASSWORD=password
     volumes:
       - postgres-data:/var/lib/postgresql/data
     healthcheck:
-      # Need to specify name/user to avoid `FATAL: role "root" does not exist` errors in logs
-      test:
-        [
-          'CMD-SHELL',
-          'env',
-          'pg_isready',
-          '--dbname',
-          '$$POSTGRES_DB',
-          '-U',
-          '$$POSTGRES_USER',
-        ]
-      interval: 10s
-      timeout: 10s
-      retries: 3
+      test: ['CMD-SHELL', 'pg_isready -U $$POSTGRES_USER -d $$POSTGRES_DB']
+      interval: 5s       # Check every 5 seconds for readiness
+      timeout: 5s        # Allow up to 5 seconds for a response
+      retries: 3         # Fail after 3 unsuccessful attempts
+      start_period: 10s  # Start checks after 10 seconds
     networks:
       - indexer-network
+
   redis:
     image: 'redis:7.2-alpine'
     restart: unless-stopped
-    command: --loglevel warning --maxmemory-policy noeviction
```

Comment:
is there a reason for removing these flags?

Reply:
Oh no, I was troubleshooting and it got left out